docs/content/management-bundles.md

```diff
@@ -680,6 +680,12 @@ Both methods are going to need a policy for either the service account or the IA
 3. Once the policy has been created, it can be assigned to the role.
 4. With the role created, go to the EC2 instance view. Select an instance where OPA will run and select "Actions" -> "Security" -> "Modify IAM role". Select the role created in previous steps.
 
+##### Web Identity Credentials
+
+Using EKS IAM Roles for Service Account (Web Identity) Credential
+
+1. Please read [IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html/) to setup.
```

Comment:
Nit: remove the trailing `/`

Reply:
It would be great if we could give an overview of the exact steps and then if needed point to the docs. Similar to the  other sections like `Metadata Credentials`.

Reply:
Updated!